infrastructure/main.tf

```diff
@@ -14,6 +14,8 @@ locals {
 
   ccdCnpUrl = "http://ccd-data-store-api-${local.local_env}.service.${local.local_ase}.internal"
 
+  document_management_url = "https://dm-store-${local.local_env}.service.${local.local_ase}.internal"
```

Comment:
keep this camelCase to be consistent

Reply:
also - i think should be http only? internal services

Reply:
sorry @srineelesh !!!! this will need to be moved from here and defined in the per env tfvars file. we only want to turn on for SPROD for now. Define a new variable with default `false` and just define url for SPROD. Once we go to production we can tidy up like this.